package.json

```diff
@@ -65,10 +65,10 @@
                     "default": "${workspaceFolder}/build.zig",
                     "description": "The path to build.zig. This is only required if zig.buildOptions = build."
                 },
-                "zig.formatCommand": {
+                "zig.zigPath": {
                     "type": "string",
                     "default": null,
-                    "description": "The command to execute zig fmt"
+                    "description": "Set a custom path to the Zig command. Defaults to 'zig' in your PATH."
```

Comment:
shouldn't this be the path to the zig compiler, not the zig command?

Reply:
This is the command used to invoke Zig. Maybe it would be clearer if it were `Set a custom path to the Zig binary`.

Reply:
Changed it to:

`Set a custom path to the Zig binary. Defaults to 'zig' in your PATH.`